### DISCUSSION TITLE
Custom syntax highlighting

### DISCUSSION THREAD

**Discussion by surli** [WeblateOrg/weblate#8583] [Q&A] [answered]
Hi, 

I'd like to know if it's possible to configure a custom syntax highlighting for specific format on Weblate?
We're using Weblate for XWiki pages translation, and we're currently discussing to integrate or not in our translations some elements of XWiki syntaxes (see: https://forum.xwiki.org/t/syntax-support-in-translations-translation-parameters/11646) However to go in that direction it would be useful for our translators that Weblate recognize those syntax elements.

**Comment by nijel** [comment, marked as answer]
The syntax highlighting in Weblate is done using https://prismjs.com/, so first you should add the format there. Integrating another highlighter into Weblate is not hard then.

**Reply by surli** [reply]
Just to be sure, I was talking about syntax highlighting in the editor here, that's what prismjs is used for in Weblate?

**Reply by nijel** [reply]
Yes.
